include/aws/crt/Api.h

```diff
@@ -59,7 +70,14 @@ namespace Aws
         };
 
         AWS_CRT_CPP_API const char *ErrorDebugString(int error) noexcept;
+        /**
+         * @return the value of the last aws error on the current thread. Return 0 if no aws-error raised before.
+         */
         AWS_CRT_CPP_API int LastError() noexcept;
+        /**
+         * @return the value of the last aws error on the current thread.
+         * @return AWS_ERROR_UNKNOWN, if no aws-error raised before.
```

Comment:
two "@return" in a comment is weird, right?

Reply:
Yep, removed.